release-engineering/platforms.md

```diff
@@ -0,0 +1,67 @@
+# Supported Platforms
```

Comment:
Could this be "Platform Builds"?

The support word is confusing.  The paragraph following this title doesn't define support.  For the context of this document it feels like "support" is meaning something around "how SIG Release and its Release Engineering subproject stand behind the artifacts they build", especially because of the repo and subdirectory in which the file sits and how it starts with "The Kubernetes release process produces artifacts..."

But it's easy to read something bigger as implied, eg: "The Kubernetes project's development process supports artifacts..."  This is a subtle but important distinction in terms of correctly setting expectations on the reader's side.

Reply:
Yes I agree, changed as suggested.